src/test/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGeneratorTest.java

```diff
@@ -71,22 +71,24 @@ public void testEiffelSourceChangeSubmittedEventGenerator() {
         setUpMocks();
         populateChangeMergedEvent();
 
-        EiffelSourceChangeSubmittedEvent eiffelEvent = EiffelSourceChangeSubmittedEventGenerator.generate(
-                changeMergedEvent, pluginConfig);
+        // EiffelSourceChangeSubmittedEvent eiffelEvent =
```

Comment:
Please activate tests or remove commented code.

Reply:
We wrote in the PR that we will add the tests so skip them for now, we will send update in this PR for the tests.

Reply:
Now the tests are in 😄